### ISSUE TITLE
why did you move from lasagne to keras?

### ISSUE THREAD

**Issue by natny** [YerevaNN/mimic3-benchmarks#49]
What improvements did you notice? I also saw that the LSTM shrank from 512 to 256.. Can you please tell me why so I don't have to go through the code in depth?

**Comment by Harhro94** [comment]
Hi @natny, the main improvement comes from Keras masking layers. The second improvement comes from the way Keras applies dropout on RNNs (notice there are two parameters `recurrent_dropout` and `dropout`). The shrinking you mentioned wasn't done intentionally, it was reached empirically.

**Comment by natny** [comment]
Thanks! How do we now use the trained model to test?

**Comment by Harhro94** [comment]
Run the same script but with the two following arguments added: `--load_state [path_to_the_model] --mode test`.

**Comment by natny** [comment]
Thanks!
